moto/secretsmanager/models.py

```diff
@@ -179,7 +187,7 @@ def create_secret(
         secret_binary=None,
         description=None,
         tags=[],
-        **kwargs
+        **kwargs,
```

Comment:
Looks like the build is failing because of this comma: https://travis-ci.org/github/spulec/moto/builds/731710916

Reply:
Thanks - fixed.